doc/architecture-decisions/0017-moving-istio-configuration-out-of-this-repo.md

```diff
@@ -0,0 +1,44 @@
+# 17. Moving Istio and Related Configuration to CF-for-K8s Repo
+
+Date: 2020-09-15
+
+## Status
+
+Accepted
+
+## Context
+
+This ADR reverts the decision made in [ADR # 7. Maintain Generated
+Istio](./0007-maintain-generated-istio.md).
+
+The networking config and related Istio config is spread widely throughout both
+cf-for-k8s and cf-k8s-networking. Having the config in both places has made
+processes such as updating networking config, versioning routecontroller,
+upgrading Istio, deciding where some networking config should exist (in this repo or
+in the cf-for-k8s repo), and so on complicated.
+
+
+## Decision
+
+Relint and Networking came to a consensus to do move Istio configuration to
+cf-for-k8s repo so we reduce the overhead of having incompatibility between
+cf-k8s-networking and cf-for-k8s.
+
+* move Istio config generation and overlays folder `istio-install`
+* move Istio generated and other networking config folders `config/istio`,
+  `config/istio-generated`
+* when contributing to networking in cf-for-k8s open PR and tag it with
+  `networking` tag
+* create CI job to run acceptance tests upon new networking PRs in cf-for-k8s
```

Comment:
I wonder if we should talk about our decision to keep NATs in our repo (and i think our pipeline in our concourse right?), or if that's a separate ADR 🤔

Reply:
I would say put it in this one. IMO the decision to keep them in our repo only means something in the context of moving istio config out of our repo